### ISSUE TITLE
Missing Documentation on Blocklist Integrations with Remediation Components

### ISSUE THREAD

**Issue by ziracmo** [crowdsecurity/crowdsec-docs#645]
Hello,

I noticed no documentation currently explaining how to integrate CrowdSec’s blocklist system into a remediation component. It would be helpful to have a dedicated section in the documentation that outlines the steps for such an integration and lists remediation components that are compatible with this system.

It could be nice to redirect to the Console integrations too from this documentation section.

Thank you!

Here are the [RC doc](https://docs.crowdsec.net/u/bouncers/intro)

**Comment by LaurenceJJones** [comment]
We currently have this document on how to create the integration

https://docs.crowdsec.net/u/integrations/remediationcomponent

but yes, there is no steps afterwards to link it back to your current RC

**Comment by ziracmo** [comment]
Yes, it's more about how I add the created link to my remediation component.
